enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -16,11 +16,9 @@ export const SdkContentWrapper = styled.div<{ font: string; baseUrl?: string }>`
   ${aceEditorStyles}
   ${saveDomImageStyles}
   ${({ theme }) => getRootStyle(theme)}
-  ${baseStyle}
```

Comment:
This only contains a bunch of CSS reset, I think it's not a bad idea to leave them out for now until we can figure a better solution.